packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -130,14 +130,10 @@ const String _kColorBackgroundWarning = 'Cannot provide both a backgroundColor a
 /// ```
 /// {@end-tool}
 ///
-/// {@tool sample}
```

Comment:
Using the `{@tool sample}` without any code results in an empty code box:
![Screen Shot 2019-07-10 at 9 38 06 AM](https://user-images.githubusercontent.com/16964204/60992711-eb54aa80-a301-11e9-829b-47266cfd0340.png)